### DISCUSSION TITLE
Selection-box to drag?

### DISCUSSION THREAD

**Discussion by LDAsh72** [Garux/netradiant-custom#58] [Q&A] [answered]
Am I missing something?  How do I drag a selection-box to select everything within?
At the moment I am using ye olde "select inside" which is much slower...

**Comment by LDAsh72** [comment, marked as answer]
Ah, I just figured it out - shift+right-click and drag!

**Reply by Garux** [reply]
May find more of these in menu.help.mouse shortcuts !

**Comment by LDAsh72** [comment]
Got a URL?  Because when I do that from NetRadiant, a popup reads "Failed to launch browser."

**Comment by Garux** [comment]
https://github.com/Garux/netradiant-custom/blob/master/docs/mouse%20shortcuts.txt or radiantFolder/docs/mouse shortcuts.txt
